proxy/http/remap/PluginDso.cc

```diff
@@ -332,3 +355,17 @@ PluginDso::LoadedPlugins::indicatePostReload(bool reloadSuccessful, const std::u
 }
 
 Ptr<PluginDso::LoadedPlugins> PluginDso::_plugins;
+
+bool
+isPluginDynamicReloadEnabled()
```

Comment:
I'm a bit concerned about this - how often is this method called? Might it not be better to have a global variable with the value from the configuration, which is updated when "records.config" is reloaded? You could look at `Enable_Config_Var`, `HttpEstablishStaticConfigByte`,  `http_server_session_sharing_cb`, or `http_config_enum_read`. The server session sharing configuration, or look at how HttpConnectionCount values are set up.

Reply:
Looking further in the code and checking the documentation, this does not appear to be reloadable itself and is therefore an event better candidate for reading once and storing in a global variable.

Reply:
Why don't we move this out of the `PluginDso` and call it once or as many times we need and pass a parameter to `PluginDso`. I think this will improve the modularity (easier to test as well, w/o the dependency on the the configuration code)

Reply:
Thanks for the response Alan. I will incorporate this and other feedback from you in my updated PR. Having a global variable will work in this case. Can you suggest the best place to define and initialize he global variable?

Reply:
fixed this.